### PR TITLE
added occupancy maps to moveit

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: robotino_bringup/robots/uibk-robotino2-sh/launch/uibk-robotino2-sh.launch [hunk 1]
```diff
@@ -8,7 +8,8 @@
 
   <!-- COMMON -->
   <!-- send robot urdf to param server -->
-  <param name="robot_description" command="$(find xacro)/xacro.py '$(find robotino_bringup)/robots/$(arg robot)/urdf/robotino.urdf.xacro'" />
```
Comment (NadiaHG) [inline robotino_bringup/robots/uibk-robotino2-sh/launch/uibk-robotino2-sh.launch:11]: please use the robotino.urdf.xacro

File: robotino_bringup/robots/uibk-robotino2-sh/urdf/robotino.urdf [hunk 2]
```diff
@@ -915,11 +915,11 @@ axis momenta are :
...
   </link>
   <!-- MOTOR JOINT -->
-  <joint name="hand_synergy_joint" type="revolute">
+  <joint name="hand_synergy_joint" type="fixed">
     <parent link="hand_palm_link"/>
     <child link="hand_invisible_wire_link"/>
-    <axis xyz="0 0 1"/>
-    <limit effort="100" lower="0.0" upper="1.0" velocity="100000"/>
+    <!-- <axis xyz="0 0 1" />
+			<limit lower="${synergy_lb}" upper="${synergy_ub}" effort="${max_synergy_effort}" velocity="${max_synergy_velocity}"/> -->
```
Comment (NadiaHG) [inline robotino_bringup/robots/uibk-robotino2-sh/urdf/robotino.urdf:922]: did you modify the file manually or running the make command?

**Comment by mzillich** [comment]
Hi there, Can this be fixed?

File: robotino_bringup/robots/uibk-robotino2-sh/urdf/robotino.urdf (hunk 2 above)
Comment (lokalmatador) [inline robotino_bringup/robots/uibk-robotino2-sh/urdf/robotino.urdf]: @smhaller do you remember how we did this?

Comment (ghost) [reply robotino_bringup/robots/uibk-robotino2-sh/urdf/robotino.urdf:922]: have to talk to @smhaller again how what exactly we did back then; could be, yes

File: robotino_bringup/robots/uibk-robotino2-sh/launch/uibk-robotino2-sh.launch (hunk 1 above)
Comment (ghost) [reply robotino_bringup/robots/uibk-robotino2-sh/launch/uibk-robotino2-sh.launch:11]: as mentioned in the mail, using the robotino.urdf.xacro results in missing links (bsaelinkx,-y,-z); using the generated urdf file (from the robotino_base_plan.urdf.xacro) results all links being there.

**Comment by ghost** [comment]
ok, this now finally should be correct - at least I hope so...

**Comment by ghost** [comment]
shell has been removed; either delete all requests or merge it to this last one - thanks.

File: robotino_bringup/robots/uibk-robotino2-sh/urdf/robotino_base_plan.urdf.xacro [hunk 3]
```diff
@@ -19,6 +20,10 @@
     </plugin>
   </gazebo>
 
+  <xacro:shell name="base_shell" parent="base_link">
```
Comment (NadiaHG) [inline robotino_bringup/robots/uibk-robotino2-sh/urdf/robotino_base_plan.urdf.xacro]: here is still the shell, please remove this

**Review by ipa-nhg — CHANGES_REQUESTED** [review]

File: robotino_bringup/robots/uibk-robotino2-sh/urdf/robotino_base_plan.urdf.xacro (hunk 3 above)
Comment (ghost) [reply robotino_bringup/robots/uibk-robotino2-sh/urdf/robotino_base_plan.urdf.xacro]: yeah, i did it on the robot, but apparently forgot to push it (and the robot's already gone); pls remind me in Freiburg again :D na, I shouldn't forget to push it.

File: robotino_bringup/robots/uibk-robotino2-sh/urdf/robotino.urdf.xacro [hunk 4]
```diff
@@ -6,11 +6,12 @@
   <xacro:include filename="$(find robotino_description)/urdf/base.urdf.xacro" />
   <xacro:include filename="$(find robotino_description)/urdf/hokuyo.urdf.xacro" />
   <xacro:include filename="$(find robotino_description)/urdf/kinect.urdf.xacro" />
+  <xacro:include filename="$(find robotino_description)/urdf/base_shell.urdf.xacro" />
   <xacro:include filename="$(find robotino_description)/urdf/pan_tilt_axis.urdf.xacro" />
   <xacro:include filename="$(find robotino_description)/urdf/arm.urdf.xacro" />
   <xacro:include filename="$(find softhand_description)/model/softhand_withCableCanal.urdf.xacro" />
 
-  <xacro:base name="base" has_8dof="false"/>
+  <xacro:base name="base" has_8dof="true"/>
```
Comment (NadiaHG) [inline robotino_bringup/robots/uibk-robotino2-sh/urdf/robotino.urdf.xacro]: please set to false see https://github.com/squirrel-project/squirrel_robotino/pull/190#issuecomment-288980177

Comment (ghost) [reply robotino_bringup/robots/uibk-robotino2-sh/urdf/robotino.urdf.xacro]: ok, seems I'm missing sth; what is this flag for? I assumed we need it for planning for both the arm and the base?

Comment (NadiaHG) [reply robotino_bringup/robots/uibk-robotino2-sh/urdf/robotino.urdf.xacro]: an URDF model with the base joints is only needed to generate the moveit configuration https://github.com/squirrel-project/squirrel_robotino/blob/indigo_dev/robotino_bringup/robots/uibk-robotino2-sh/urdf/robotino_base_plan.urdf.xacro , if you load an URDF with 8 dof as robot_description, the urdf model will limit the movements of the base, the problem that reported Senka.

Comment (ghost) [reply robotino_bringup/robots/uibk-robotino2-sh/urdf/robotino.urdf.xacro]: ah, I see; kudos! thanks for clarifying that!

**Comment by ghost** [comment]
fixed, pushed and requested to merge!

**Review by ipa-nhg — APPROVED** [review]

**Pull request merged**